src/app/campaigns/campaign-edit/campaign-edit.component.ts

```diff
@@ -83,7 +91,7 @@ export class CampaignEditComponent {
             from: this.updateForm.value.returnFrom,
             to: this.updateForm.value.returnTo,
           },
-          newsURLs: this.updateForm.value.newsURLs,
+          newsURLs: this.newsUrls.value.map((fg: any) => fg.url),
```

Comment:
1. maybe ensure not to cal `map()` on the value if value is undefined (`newsUrls.value?.map()`)?
2. fg? like form group? can you call it item or something?